concepticondata/references/references.bib

```diff
@@ -3090,3 +3090,20 @@ @article{Alonso2018
   year={2018},
   publisher={Cambridge University Press}
 }
+
+@misc{Hsiu2015,
```

Comment:
@macyl, don't use this bibtex, but follow the one I provided, or label it as inproceedings:

```
@misc{Hsiu2015,
  author = {Hsiu, Andrew C.},
  title = {...},
  year = {...},
  howpublished = {Paper presented at SEALS 25},
  venue = {Chiang Mai},
  eventdate = {2015-05-27/2015-05-29},
  address = {Bangkok},
  publisher = {...},
  doi = {...},
  url = {...}
}
```

Glottolog is *never* to be pasted literatlly to concepticon, as the references do not follow our scheme. 
`...` indicats: follow the value I gave you.